### PR TITLE
Updated to latest bootstrap, jquery, filesaver.

### PR DESCRIPTION
**Pull request by jasoncoon**
* Updated to latest bootstrap, jquery, filesaver.
* Switched from glyphicons to bootstrap icons.
* This will allow us to switch to "vanilla" JS & remove jquery.
* Hide the import/export accordion until the app is loaded.
* Use enableCORS, remove individual CORS headers.

The look of the web app didn't change drastically.

Before:

image

After:

image

### CODE REVIEW

**Comment by henrygab** [comment]
Hi Jason,

The UI looks *much* cleaner.  Well done!

**Q:** Is there any reason not to just use the version that is on the local file system?  I also put my LEDs on an isolated network, and in many places, people pay for all data transferred (no flat rate internet).

File: esp8266-fastled-webserver/data/js/simple.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // used when hosting the site on the ESP8266
-var address = location.hostname;
-var urlBase = "";
+// var address = location.hostname;
+// var urlBase = "";
 
 // used when hosting the site somewhere other than the ESP8266 (handy for testing without waiting forever to upload to SPIFFS/LittleFS)
-// var address = "192.168.1.13";
-// var urlBase = "http://" + address + "/";
+var address = "192.168.86.30";
```
Comment (henrygab) [inline esp8266-fastled-webserver/data/js/simple.js:6]: I think this was left in by accident?

**Review by henrygab — CHANGES_REQUESTED** [review]
I'm going to expose my ignorance about HTML/JS coding....

1. See comment ... debugging data left in accidentally?
2. Are all five versions of the `glyphicons` file required?

**Comment by jasoncoon** [comment]
> The UI looks much cleaner. Well done!

Thanks!

> **Q:** Is there any reason not to just use the version that is on the local file system? I also put my LEDs on an isolated network, and in many places, people pay for all data transferred (no flat rate internet).

Yes, the local file system static files are not currently being served correctly. See https://github.com/jasoncoon/esp8266-fastled-webserver/issues/231#issuecomment-997466407

File: esp8266-fastled-webserver/data/js/simple.js (hunk 1 above)
Comment (jasoncoon) [reply esp8266-fastled-webserver/data/js/simple.js]: 1. Yes, fixed, thanks!
2. All of the glyphicons files have been removed as part of this PR.

**Review by henrygab — APPROVED** [review]
Note: my review is cursory at best, due to my limited expertise in the HTML/JS.

**Pull request merged**